src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" />
     <PackageReference Include="OpenIddict" />
+    <PackageReference Include="System.Security.Cryptography.Cng" Version="$(SystemSecurityCryptographyCng)" />
```

Comment:
Remove `Version="$(SystemSecurityCryptographyCng)"` ?

Reply:
The other `System.*` dependency present in OC was defined the same way (see `SystemComponentModelAnnotations`). I can certainly change that to make them consistent with the ASP.NET Core packages.

Reply:
Yes please. Missed the other one then.

Reply:
Done.